omnibus/resources/chef/msi/source.wxs.erb

```diff
@@ -78,6 +87,13 @@
         </Directory>
         <Directory Id="INSTALLLOCATION" Name="opscode">
           <Directory Id="PROJECTLOCATION" Name="$(var.ProjectLocationDir)" >
+            <Component Id="ProjectLocationPermissions" Guid="{75f50556-efae-4ede-beb2-a2c9b1a4d43f}" >
```

Comment:
Maybe worth a note describing why this is needed and/or what happens instead if this section isn't present.